### ISSUE TITLE
Computational resources?

### ISSUE THREAD

**Issue by ghost** [zhao-ht/GIMLET#1]
Hi, thank you sharing your great work!
It is quite interesting to use molecular structure as well as text information.
I wonder that such a language model seems require heavy computational resources.
How rich GPUs are required to pretrain the model when I want to try your work.
Thank you in advance:)

**Comment by zhao-ht** [comment]
Thank you for your interest in our work!
We utilize the T5-small model as our backbone, which consists of 64 million parameters. Pretraining can be effectively performed on a single 32 GB GPU. Additionally, you have the option to employ gradient accumulation to further mitigate memory requirements.

**Comment by zhao-ht** [comment]
Thank you for your interest in our work!
We utilize the T5-small model as our backbone, which consists of 64 million parameters. Pretraining can be effectively performed on a single 32 GB GPU. Additionally, you have the option to employ gradient accumulation to further mitigate memory requirements.

**Comment by ghost** [comment]
@zhao-ht Thank you for your kind reply!
That’s awesome only a single GPU is required for pretraining.
I appreciate also for the helpful option.
I will try it:)
